src/components/FlightsTable/FlightsTable.jsx

```diff
@@ -7,9 +7,9 @@ export const FlightsTable = ({originFilter, flights}) => {
     <section className="flightsTable">
       <h3>Flights</h3>
       <section>
-        {flights.filter((flight) => (
+        {flights?.filter((flight) => (
           originFilter.length ? originFilter.includes(flight.origin) : flight
-        )).map((flight) => (
+        ))?.map((flight) => (
```

Comment:
If `flights` is not undefined then this will never be undefined. No need to throw an extra null check here.

Reply:
Good point, fixed